File: app/src/main/java/omniex/nl/omniex/ui/app/order/shipping/method/OrderShippingMethodView.java
package omniex.nl.omniex.ui.app.order.shipping.method;

import java.util.List;

import nl.omniex.omniexshopping.data.model.shipping.ShippingMethod;
import nl.omniex.omniexshopping.ui.base.BaseView;

public interface OrderShippingMethodView extends BaseView {

    void onShippingMethodsFetched(List<ShippingMethod> shippingMethods);
    void onShippingMethodSet();
}

File: app/src/main/java/omniex/nl/omniex/ui/app/order/shipping/method/OrderShippingMethodView.kt
package omniex.nl.omniex.ui.app.order.shipping.method

import omniex.nl.omniex.data.model.shipping.ShippingMethod
import omniex.nl.omniex.ui.base.BaseView

interface OrderShippingMethodView : BaseView {

    fun onShippingMethodsFetched(shippingMethods: List<ShippingMethod>)
    fun onShippingMethodSet()
}
